pkg/rke/templates/templates.go

```diff
@@ -116,9 +117,10 @@ var TemplateIntroducedRanges = map[string][]string{
 func LoadK8sVersionedTemplates() map[string]map[string]string {
 	return map[string]map[string]string{
 		kdm.Calico: {
-			">=1.27.0-rancher1-1":                      calicov3261,
-			">=1.26.0-rancher1-1 <1.27.0-rancher1":     calicov3250,
-			">=1.25.0-rancher1-1 <1.26.0-rancher1-1":   calicov3241,
+			">=1.26.11-rancher1":                       calicov3261,
```

Comment:
Are we positive we're not going to hit any issues with upgrade given how templates here reference v3.26.1 vs. v3.26.3 in 2.7 PR (https://github.com/rancher/kontainer-driver-metadata/pull/1257/files#diff-98a74f75bbfd8f2ab7594591f9e71d268da7423281099e275c61bf283848928fR119)? I understand the templates may be the same but I'm concerned about names mismatch.

The same applies to Canal.

@kinarashah

Reply:
updated the 2.7 PR.... no changes needed in this PR

Reply:
@kinarashah , @vardhaman22 , thank you!